src/cript/api/api.py

```diff
@@ -217,7 +193,21 @@ def host(self):
         https://criptapp.org/api/v1
         ```
         """
-        return self._host
+        host = self._raw_host.strip()
```

Comment:
how come this was removed from `_perpare_host` function and put here?

Reply:
And extra function for this seems overkill.
I get, why you did it, since you only need to construct it once.
But I liked it being more dynamic.

Reply:
Also, since we are now using member attributes of the API class for the host construction.
I either had to pass an API object to the `_prepare_host` function, or the prefix separately.
And passing an API object into the function, is basically the same as a member function.
So, I made it a member function right away.
And to no clunk up the class namespace anymore then necessary, the host property seems a good place for it.

Reply:
but it's still only constructed once, how come host needs to be dynamic?

Reply:
no, now it is constructed every time you call `API.host`

Reply:
how come we don't want to construct it once and save it in the class as a class variable?

Reply:
I strongly recommend having everything for the host within `_prepare_host` where it was encapsulated nice and neatly

Reply:
That is a possible solution too.
But it needs one more function to debug.
My feeling was that the namespace was pretty full already, so I got rid of one function.

Reply:
> I strongly recommend having everything for the host within `_prepare_host` where it was encapsulated nice and neatly.

I disagree here.
You encapsulated it too much.
When I needed access to the API version string and where it was constructed, it took me a while to find it.

Reply:
I gave you a solution that just put the hard coded string into a global variable.
Like anything hard coded should be.
But you didn't like that approach either.